### PR TITLE
Visual hints

### PR DESCRIPTION
**Pull request by rohankatyal29**
The user can see visual hints on google images before seeing the actual translation of the word. This will keep the user engaged and will keep him guessing. This will help the user in remembering the translations. 

screen shot 2016-07-12 at 4 02 46 pm

screen shot 2016-07-12 at 4 04 02 pm

### CODE REVIEW

File: lib/scripts/eventPage.js [hunk 1]
```diff
@@ -148,27 +141,29 @@ function setup() {
     'contexts': ['selection'],
     'id': 'removeFromDifficultyBucket'
   });
+  chrome.contextMenus.create({
+    'title': 'Visual Hints',
```
Comment (Ceilican) [inline lib/scripts/eventPage.js]: Could you change this to "Show Visual Hints"?

File: lib/scripts/eventPage.js [hunk 2]
```diff
@@ -238,127 +231,114 @@ function contextMenuClickHandler(info, tab) {
     active: true
   }, (tabs) => {
     var tabURL = tabs[0].url;
-    switch (info.menuItemId) {
```
Comment (Ceilican) [inline lib/scripts/eventPage.js]: What was the motivation to change the switch statement to a series of if statements? The switch statement seemed less verbose.

File: lib/scripts/eventPage.js (hunk 1 above)
Comment (Ceilican) [inline lib/scripts/eventPage.js]: Could you also change the few other menu titles (e.g. "SpeakTheWord", ) that do not conform to the linguistic style of most other titles?

**Comment by Ceilican** [comment]
Thanks!

File: lib/scripts/eventPage.js (hunk 2 above)
Comment (rohankatyal29) [inline lib/scripts/eventPage.js]: @ankit-m: didn't you make these changes?

**Pull request merged**